ibi/coreos-installer/scripts/ostree-restore.sh

```diff
@@ -3,97 +3,12 @@
-        return 0
-    elif [ "$(stat -c %d /sysroot)" != "$(stat -c %d /var/lib/containers)" ]; then
-        # Container storage is on a separate partition from /sysroot
-        return 0
-    fi
-
-    return 1
-}
-
-if [[ -z "$seed_image" ]]; then
-    echo "ERROR. Seed image is empty"
-    exit 1
-fi
-
-# mount /sysroot -o remount,rw
-
-# Import OCIs
-log_it "Pulling and mounting backup OCI"
-podman pull --authfile="/var/tmp/backup-secret.json" ${seed_image}
-img_mnt=$(podman image mount ${seed_image})
-
-# Collect seed deployment data from the backup
-upg_booted_id=$(jq -r '.deployments[] | select(.booted == true) | .id' ${img_mnt}/rpm-ostree.json)
-upg_booted_deployment=${upg_booted_id/*-}
-upg_booted_ref=${upg_booted_deployment/\.*}
-
-up_ver=$(jq -r '.version' ${img_mnt}/manifest.json)
-new_osname=rhcos_${up_ver}
-
-ostree_repo=$(mktemp -d -p /mnt/ostree/deploy/rhcos/var/tmp)
-
-tar xzf $img_mnt/ostree.tgz --selinux -C $ostree_repo
-
-# We need to adjust to the mounted disk ostree repo path
-log_it "Importing remote ostree"
-ostree pull-local --repo /mnt/ostree/repo $ostree_repo
-ostree admin os-init --sysroot /mnt $new_osname
-
-log_it "Creating new deployment $new_osname"
-# We should create the new deploy as not-default, and after the whole process is done, be able to switch to it for the next reboot
-# ostree admin deploy --os ${new_osname} $(build_kargs) --not-as-default ${upg_booted_ref}
-# Until I find how to do the switch, I'll deploy as default
-
-ostree admin deploy --sysroot /mnt --os ${new_osname} $(build_kargs) ${upg_booted_ref}
-ostree_deploy=$(ostree admin status --sysroot /mnt | awk /$new_osname/'{print $2}')
-
-# Restore the seed .origin file
-cp ${img_mnt}/ostree-${upg_booted_deployment}.origin /mnt/ostree/deploy/$new_osname/deploy/$ostree_deploy.origin
-
-log_it "Restoring /var"
-tar xzf ${img_mnt}/var.tgz -C /mnt/ostree/deploy/$new_osname --selinux
-
-log_it "Restoring /etc"
-tar xzf ${img_mnt}/etc.tgz -C /mnt/ostree/deploy/$new_osname/deploy/$ostree_deploy --selinux
-
-log_it "Removing /etc deletions"
-cat ${img_mnt}/etc.deletions | xargs --no-run-if-empty -ifile rm -f /mnt/ostree/deploy/$new_osname/deploy/$ostree_deploy/file
-
-log_it "Precaching container images"
-xargs --no-run-if-empty --max-args 1 --max-procs 10 crictl pull < ${img_mnt}/containers.list
+seed_version=${2:-$SEED_VERSION}
+authfile=${AUTH_FILE:-"/var/tmp/backup-secret.json"}
+image=${3:-$LCA_IMAGE}
+pull_secret=${PULL_SECRET_FILE:-"/var/tmp/pull-secret.json"}
 
-# log_it "Unpacking certificates to be used by recert"
-# certs_dir=/mnt/ostree/deploy/$new_osname/var/opt/openshift/certs
-# mkdir -p $certs_dir
-# tar xpvzf /tmp/certs.tgz -C $certs_dir
+sudo podman run --privileged --rm --pid=host --authfile "${authfile}" -v /:/host --entrypoint /usr/local/bin/ibu-imager "${image}" ibi --seed-image "${seed_image}" --authfile "${authfile}" --seed-version "${seed_version}" --pullSecretFile "${pull_secret}"
```

Comment:
Move this to the other script and ditch ostree-backup.sh, since ostree-backup.sh was suposed to be a temporary workaround until we could replace it with LCA